### PR TITLE
Header

### PR DESCRIPTION
**Pull request by aniaorwatk**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import react, {FC} from 'react';
+import Container from '@mui/material/Container';
```
Comment (paulinaTargosz) [inline src/components/Header/Header.tsx]: Move Container to Box i Typography - line below

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import react, {FC} from 'react';
+import {Box, Typography} from '@mui/material';
+import './Header.css'
+
+const Header: FC = () => {
+  const title: string = 'Meet ReqRes users!'
+  const subtitle: string = 'Application created with free ReqRes API'
+
+    return(
```
Comment (paulinaTargosz) [inline src/components/Header/Header.tsx:9]: fix formatting here

File: src/index.css [hunk 3]
```diff
@@ -11,3 +11,5 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+
```
Comment (paulinaTargosz) [inline src/index.css]: why?

Comment (aniaorwatk) [reply src/index.css]: deleted

**Review by paulinaTargosz — APPROVED** [review]

**Pull request merged**